configs/14.0/release_notes/release_notes-inst.html

```diff
@@ -289,6 +289,39 @@ <h3>Using aptitude or apt</h3>
 				<p>Aptitude and apt support package upgrades for new revision releases (i.e. 10.0-0 to 10.0-1).  For new major releases, install as if a new installation.</p>
 endif_ubuntu-18 -->
 
+<!-- if_ubuntu-20
+                        <p>The gpg public key <span class="code">xxxxxxxx-gpg.key</span>will be provided in the repository where these release notes were found.  The public key can be used to verify the authenticity of both the Advance Toolchain install packages and the repository contents. After downloading the gpg public key, it can be imported as follows:</p>
+                        <p class="code">sudo apt-key add xxxxxxxx.gpg.key</p>
+                        <h3>Using aptitude or apt</h3>
+                                <p>If the Advance Toolchain repository has not already been configured, that must be done first.  As <i>root</i>, edit the following file, adding one of the lines as needed for the installed system:
+                                <p class="code">/etc/apt/sources.list</p>
+                                <ul>
+                                        <li>For amd64 workstations or POWER servers (ppc64le) add the line:</li>
+                                        <p class="code">deb ftp://public.dhe.ibm.com/software/server/POWER/Linux/toolchain/at/ubuntu focal __VERSION__ </p>
```

Comment:
I've recently seen an error message on Ubuntu 20.04 complaining that FTP was not supported anymore.

Reply:
Fixed